extensions/pipe-commands/assets/commands/text/convert-newline-to-space.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# @raycast.schemaVersion 1
+# @raycast.title Convert Newline to Spaces
+# @raycast.packageName Text Actions
+# @raycast.mode pipe
+# @raycast.argument1 {"type": "text", "placeholder": "Multiline text to reverse"}
+# @raycast.icon 🔤
+
+echo "$1" | tr '\n' ' '
```

Comment:
pip commands read from stdin, so this will not work. Try this instead

```
#!/bin/bash

# @raycast.schemaVersion 1
# @raycast.title Convert Newline to Spaces
# @raycast.packageName Text Actions
# @raycast.mode pipe
# @raycast.icon 🔤

tr '\n' ' '
```

Reply:
weird, this was not working for me. Let me give it another shot.